token-lending/program/src/processor.rs

```diff
@@ -2126,6 +2134,20 @@ fn get_pyth_price(pyth_price_info: &AccountInfo, clock: &Clock) -> Result<Decima
         LendingError::InvalidOracleConfig
     })?;
 
+    let conf = pyth_price.agg.conf;
+
+    let confidence_ratio: u64 = 10;
+    // Perhaps confidence_ratio should exist as a per reserve config
+    // 100/confidence_ratio = maximum size of confidence range as a percent of price
+    // confidence_ratio of 10 filters out pyth prices with conf > 10% of price
+    if conf.checked_mul(confidence_ratio).unwrap() > price {
+        msg!(
+            "Oracle price confidence is too wide: {}",
+            pyth_price.agg.status as u8
```

Comment:
Isn't this always just going to be `pyth::PriceStatus::Trading`?

Seems better to log conf and price

Reply:
ah yeah meant to do that